templates/site/index.html

```diff
@@ -0,0 +1,209 @@
+
+                <div id="searchResultsNavigation" class="search-results-nav">
+                    <div>
+                        {% with %}
+                            {% set button_type = "button_lg_nav_previous" %}
+                            {% set button_id = "searchResultsPrevious" %}
+                            {% include "partials/button.html" %}
+                        {% endwith %}
+
+                        {% with %}
+                            {% set button_type = "button_sm_nav" %}
+                            {% set button_label = "First" %}
+                            {% set button_id = "searchResultsFirst" %}
+                            {% include "partials/button.html" %}
+                        {% endwith %}
+                    </div>
+                    <div>
+                        {% with %}
+                            {% set button_type = "button_lg_nav_next" %}
+                            {% set button_id = "searchResultsNext" %}
+                            {% include "partials/button.html" %}
+                        {% endwith %}
+                    </div>
+
+                </div>
+            </div>
+
+            <div class="row">
+                <div class="container-fluid">
+                    <div id="searchResults"></div>
+                    <div id="datasetDetails">
+                        <div id="datasetContainer" class="div-container-dataset">
+                            <div id="datasetCollectionContainer">
+                                <div id="datasetCollection"><h4>Collection</h4></div>
+                                <div id="datasetCollectionName"></div>
+                            </div>
+
+                            <div id="datasetTitleContainer">
+                                <div id="datasetTitle"><h4>Dataset</h4></div>
+                                <div id="datasetName"></div>
+                            </div>
+
+                            <div id="datasetAssetsContainer">
+                                {% with %}
+                                    {% set dropdown_type = "dropdown_list_panel" %}
+                                    {% set dropdown_button_id = "datasetAssetsButton" %}
+                                    {% set dropdown_list_panel_link_id = "datasetAssetsLabel" %}
+                                    {% set dropdown_list_id  = "datasetAssetsList"%}
+                                    {% set panel_expanded = "false" %}
+                                    {% set dropdown_show = "" %}
+                                    {% set dropdown_label_text = "Assets" %}
+                                    {% include "partials/dropdown-panel.html" %}
+                                {% endwith %}
+                            </div>
+                            <div id="datasetGeneralContainer"></div>
+
+                            <div id="datasetGeometryContainer">
+                                <div id="datasetMapContainer" class="map"></div>
```

Comment:
The map doesn't render properly here:

![image](https://github.com/user-attachments/assets/b5a4e082-b3df-4cf3-a60f-3f84f4411958)